README.md

```diff
@@ -1 +1,20 @@
-# Cproject
\ No newline at end of file
+# C project ( C for College)
+
+This is a feedback system project. Through this project, we will improve the outcome of the feedback's given by the
+students for their teachers.
+
+There are many drawbacks of the traditional system . Such as, some of the teachers were able to read that which feedback is given by whom.
+
+So, to improve this we are developing a feedback system  in
+which even the admin will only be able to see the outcome of the feedback was given by students for the respective teachers.
```

Comment:
Consider this line as " in which only the admin will be able to ..."

Reply:
I want to say this 
 even admin is not able to see which student has given the particular feedback.
Which will make it more secure for students